chaco/lod_image_plot.py

```diff
@@ -0,0 +1,322 @@
+from contextlib import contextmanager
+
+from enable.base import intersect_bounds
+from traits.api import (
+    Event, Instance, Int, List, Property,
+    cached_property, on_trait_change
+)
+try:
+    # InterpolationQuality required for Quartz backend only (requires OSX).
+    from kiva.quartz.ABCGI import InterpolationQuality
+except ImportError:
+    pass
+else:
+    KIVA_INTERP_QUALITY = {"nearest": InterpolationQuality.none,
+                           "bilinear": InterpolationQuality.low,
+                           "bicubic": InterpolationQuality.high}
+try:
+    from encore.concurrent.futures.serializer import Serializer
+    from encore.concurrent.futures.enhanced_thread_pool_executor import \
+        EnhancedThreadPoolExecutor
+except ImportError:
+    import sys
+    sys.exit('encore must be installed to use LODImagePlot')
```

Comment:
You shouldn't do this if you're going to import from this module in `chaco.api`.

(disregard if encore is already a hard requirement)

Reply:
I realized that after seeing the CI failed. `encore` is not a hard requirement ATM.

Reply:
Do you know any better place to import this module from? Are there anything like `chaco.experimental.api`?

Reply:
No. I think the answer it to just leave it out and force users to import it directly.

Reply:
Moved as suggested.